appcues/src/main/java/com/appcues/debugger/DebuggerStatusManager.kt

```diff
@@ -288,44 +315,65 @@ internal class DebuggerStatusManager(
-            if (deepLinkConfigured != null) {
-                deepLinkConfigured = null
-                updateData()
-            }
+        val manifestConfigured = context.packageManager.resolveActivityCompat(intent, PackageManager.MATCH_DEFAULT_ONLY) != null
 
-            val token = "verify-${UUID.randomUUID()}"
-            val intent = Intent(Intent.ACTION_VIEW).apply {
-                data = Uri.parse("appcues-${appcuesConfig.applicationId}://sdk/debugger/$token")
-                flags = Intent.FLAG_ACTIVITY_NEW_TASK
-            }
+        if (manifestConfigured) {
+            deepLinkValidationToken = token
+            context.startActivity(intent)
+
+            // a new link should have come in and updated our state in the checkDeepLinkValidation function above
+            // we give that a little time to process
+            delay(1.seconds)
 
-            val manifestConfigured = context.packageManager.resolveActivityCompat(intent, PackageManager.MATCH_DEFAULT_ONLY) != null
+            // if after 1 second we still do not have that updated state - it failed
+            if (deepLinkValidationToken != null) {
+                deepLinkValidationToken = null
+                deeplinkStatus = ERROR
+                deepLinkErrorText = contextWrapper.getString(R.string.appcues_debugger_status_check_deep_link_error_handler)
+                updateData()
+            }
+        } else {
+            deeplinkStatus = ERROR
+            deepLinkErrorText = contextWrapper.getString(R.string.appcues_debugger_status_check_deep_link_error_manifest)
+            updateData()
+        }
+    }
 
-            if (manifestConfigured) {
-                deepLinkValidationToken = token
-                context.startActivity(intent)
+    suspend fun checkPushValidation() {
+        if (pushStatus == SUCCESS) return
 
-                // a new link should have come in and updated our state in the checkDeepLinkValidation function above
-                // we give that a little time to process
-                delay(1.seconds)
+        // set to null and update data so it will update to loading
+        pushErrorText = null
+        pushValidationToken = null
+        pushStatus = LOADING
+        updateData()
 
-                // if after 1 second we still do not have that updated state - it failed
-                if (deepLinkValidationToken != null) {
-                    deepLinkValidationToken = null
-                    deepLinkConfigured = false
-                    deepLinkErrorText = contextWrapper.getString(R.string.appcues_debugger_status_check_deep_link_error_handler)
-                    updateData()
-                }
-            } else {
-                deepLinkConfigured = false
-                deepLinkErrorText = contextWrapper.getString(R.string.appcues_debugger_status_check_deep_link_error_manifest)
+        // create new random id and send it to the server
+        val token = "push-${UUID.randomUUID()}"
```

Comment:
It worked ok to have a non-uuid string here with a `push-` prefix?

Reply:
yes, also decided to prefix with test-push and test-deeplink for readability.